application/src/test/java/org/togetherjava/tjbot/features/basic/SlashCommandEducatorTest.java

```diff
@@ -60,11 +60,27 @@ void ignoresOtherMessages(String message) {
         verify(event.getMessage(), never()).replyEmbeds(any(MessageEmbed.class));
     }
 
+    @ParameterizedTest
+    @MethodSource("provideOtherCommands")
+    void ignoresMessagesWhereLengthIsGreaterThanThirty(String message) {
```

Comment:
i dont think its useful to add a third method in this case. just add the case to the existing `provideOtherMessages`, thanks